src/app/components_ngrx/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -213,7 +220,14 @@ export class WorkItemQuickAddComponent implements OnInit, OnDestroy, AfterViewIn
     }
   }
 
-
+  getInfotipText(id: string) {
```

Comment:
This function is repeatedly written in two components. Can we write it in one place?